src/ordered_set.jl

```diff
@@ -30,9 +30,10 @@ pop!(s::OrderedSet, x, deflt) = pop!(s.dict, x, deflt) == deflt ? deflt : x
 delete!(s::OrderedSet, x) = (delete!(s.dict, x); s)
 
 getindex(x::OrderedSet,i::Int) = x.dict.keys[i]
-endof(x::OrderedSet) = endof(x.dict.keys)
+lastindex(x::OrderedSet) = lastindex(x.dict.keys)
 # Needed on 0.7 to mimic array indexing.
```

Comment:
Which line is this about?
`lastindex`  or `nextind`
we should either specify in the comment, or move the comment onto the same line as the line it is about